npm install fails in MacOS Catalina v10.15.3

## Description

`npm install` fails in my machine due to conflicts with `node-gyp`. 

## Environment

- Compiler version: N/A
- Compiler options (if applicable, e.g. optimizer enabled): N/A
- Framework/IDE (e.g. Truffle or Remix): N/A
- EVM execution environment / backend / blockchain client: N/A
- Operating system: MacOS Catalina v10.15.3

- Node version: v13.11.0
- npm version: 6.13.7
- safe-contracts commit: aa0f3345b609a816ace6c448960ddb852b8a1bbd

## Steps to reproduce

```
$ git clone https://github.com/gnosis/safe-contracts.git
$ cd safe-contracts
$ npm install
```

Any dependency using `node-gyp` halts the installation. For example:
```
npm ERR! keccak@1.4.0 rebuild: `node-gyp rebuild`
```

## Additional context

I fixed the issue by using `npm-check-updates` to upgrade dependencies:

```
$ npm install -g npm-check-updates
$ npm-check-updates -u
$ npm install
```

See below the list of upgrades produced by `npm-check-updates`:

```
dotenv                               ^8.0.0  →   ^8.2.0 
 openzeppelin-solidity                ^2.0.0  →   ^2.5.0 
 solc                                 ^0.5.9  →   ^0.6.4 
 truffle-hdwallet-provider      0.0.7-beta.1  →   1.0.17 
 @gnosis.pm/truffle-nice-tools        ^1.3.0  →   ^1.3.1 
 @openzeppelin/cli                    ^2.5.1  →   ^2.7.2 
 @openzeppelin/upgrades               ^2.5.1  →   ^2.7.2 
 eth-lightwallet                      ^3.0.1  →   ^4.0.0 
 ethereumjs-abi                       ^0.6.5  →   ^0.6.8 
 ethereumjs-util                      ^6.0.0  →   ^6.2.0 
 ganache-cli                          ^6.3.0  →   ^6.9.1 
 run-with-testrpc                     ^0.3.0  →   ^0.3.1 
 solidity-coverage                    ^0.4.2  →   ^0.7.2 
 truffle                             ^4.1.14  →  ^5.1.18 
 web3                                 0.20.6  →    1.2.6 
````

--- a/README.md
+++ b/README.md
@@ -6,6 +6,9 @@ Gnosis Safe Contracts
 
 Install
 -------
+### The repository requires Node 10
+- As this repository uses an [old version of web3js](https://github.com/ethereum/web3.js/releases/tag/v0.20.6) it is required to use Node 10 else `npm install` will fail.
+
 ### Install requirements with npm:
 
 ```bash

